UI/ui_mainmode.csv

```diff
@@ -5662,7 +5662,7 @@ Achievement_Down#0,"""を達成しました！"""
 Clear_Msg#0,"""チャレンジクリア率：MISSION$(0) $(1)%"""
 TextStart#0,"""Challenge  start"""
 TextClear#0,"""Challenge clear"""
-TextMission#0,"""mission"""
+TextMission#0,"""Mission"""
```

Comment:
you need to keep this to the JP Text `mission` since the EN Text `Mission` will apply in the patches

Reply:
ok, i am sorry.... ><

Reply:
Done